pegasus/sites.v3/code.org/views/dance_tutorial_section.haml

```diff
@@ -0,0 +1,26 @@
+%div{style: "display: inline; float: right"}
+  %a{href: "https://code.org/translate/dance"}
+    %div{style: "color: rgb(0, 173, 188); font-size: 14px; margin-top: -2px; display: inline;"}
+      =I18n.t(:hoc2018_dance_party_subheading)
+  %a.dance-chevron-icon{href: "https://code.org/translate/dance"}
+    %i{class: "fas fa-chevron-right", style: "display: inline; color: rgb(0, 173, 188); font-size: 10px; font-weight: bold; margin-left: 15px;"}
+
```

Comment:
extract the inline styles please. you have a style sheet for this page; let's use it.